src/main/java/org/folio/circulationbff/service/impl/CirculationBffServiceImpl.java

```diff
@@ -35,4 +38,17 @@ public AllowedServicePoints getAllowedServicePoints(AllowedServicePointParams pa
       return circulationClient.allowedServicePoints(params);
     }
   }
+
+  @Override
+  public Request createRequest(BffRequest request, String tenantId) {
+    log.info("createRequest:: request: {}", request.getId());
+    if (settingsService.isEcsTlrFeatureEnabled(tenantId) && userTenantsService.isCentralTenant(tenantId)) {
+      log.info("createRequest:: ECS TLR Feature is enabled. Creating ECS TLR");
+      EcsTlr tlrRequest = ecsTlrClient.createRequest(request);
+      return circulationClient.getRequestById(tlrRequest.getPrimaryRequestId());
```

Comment:
Please make sure that we have permission for this

Reply:
Permission exists